qplaintexteditsearchwidget.h

```diff
@@ -44,7 +46,10 @@ class QPlainTextEditSearchWidget : public QWidget {
     int _currentSearchResult;
     QList<QTextEdit::ExtraSelection> _searchExtraSelections;
     QColor selectionColor;
+    QTimer _debounceTimer;
+    QString searchTerm_;
```

Comment:
Is there a reason for the trailing `_` instead of a leading one?

Reply:
No, will change it.

Reply:
Now both are trailing instead of leading...